### PR TITLE
Fix tsv downloads

### PR DESCRIPTION
**Pull request by UrsaStutsman**
- [ ] **Submitter**: Include the JIRA issue number in the PR description
- [ ] **Submitter**: Check documentation and code comments. Add explanatory PR comments if helpful.
- [ ] **Submitter**: If you changed a URL that is used elsewhere (e.g. in an email), comment about where it is used and ensure the dependent code is updated.
- [ ] **Submitter**: JIRA ticket checks:
  * Acceptance criteria exists and is met
  * Note any changes to implementation from the description
  * To Demo flag is set
  * Release Summary is filled out, if applicable
  * Add notes on how to QA
- [ ] **Submitter**: Update RC_XXX release ticket with any config or environment changes necessary
- [ ] **Submitter**: Update FISMA documentation if changes to:
  * Authentication
  * Authorization
  * Encryption
  * Audit trails
- [ ] **Submitter**: If you're adding new libraries, sign us up to security updates for them
* Review cycle:
  * LR reviews
  * Rest of team may comment on PR at will
  * **LR assigns to submitter** for feedback fixes
  * Submitter rebases to develop again if necessary
  * Submitter makes further commits. DO NOT SQUASH
  * Submitter updates documentation as needed
  * Submitter **reassigns to LR** for further feedback
- [ ] **TL** sign off
- [ ] **LR** sign off
- [ ] **Product Owner** sign off
- [ ] **Submitter**: Verify all tests go green, including CI tests
- [ ] **Submitter**: Squash commits and merge to develop
- [ ] **Submitter**: Delete branch after merge
- [ ] **Submitter**: **Test this change works on dev environment after deployment**. YOU own getting it fixed if dev isn't working for ANY reason!
- [ ] **Submitter**: Mark JIRA issue as resolved once this checklist is completed

### CODE REVIEW

File: src/cljs/main/broadfcui/page/workspace/data/tab.cljs [hunk 1]
```diff
@@ -131,13 +132,9 @@
                  :value (utils/get-access-token)}]
         [:input {:type "hidden"
                  :name "attributeNames"
-                 :value (->> (persistence/try-restore
-                              {:key (str (common/workspace-id->string
-                                          workspace-id) ":data:" selected-entity-type)
-                               :initial (constantly {})})
-                             :column-meta
+                 :value (->> (or (:visible-columns @state) ((@refs "entity-table") :get-ordered-columns))
```
Comment (MattPutnam) [inline src/cljs/main/broadfcui/page/workspace/data/tab.cljs]: I think we shouldn't keep a copy of the column state in this component, and just always ask for it via `:get-ordered-columns` when needed here.

File: src/cljs/main/broadfcui/page/workspace/data/tab.cljs [hunk 2]
```diff
@@ -131,13 +132,9 @@
...
                              (filter :visible?)
-                             (map :header)
+                             (map (some-fn :id :header))
```
Comment (MattPutnam) [inline src/cljs/main/broadfcui/page/workspace/data/tab.cljs]: I was wrong--this should just be `(map :id)`. The table uses the `:id` property, and can use `:header` if it's a string and `:id` isn't provided. But by this point, it's resolved that and just refers to columns by `:id`.

File: src/cljs/main/broadfcui/page/workspace/data/tab.cljs (hunk 1 above)
Comment (dmohs) [reply src/cljs/main/broadfcui/page/workspace/data/tab.cljs]: I agree with Putnam's suggestion. I have one more, which I will suggest as a PR.

File: src/cljs/main/broadfcui/common/entity_table.cljs [hunk 3]
```diff
@@ -49,6 +49,9 @@
                         (f selected-entity-type))
                       (after-update #(this :update-data reinitialize?)))
                     (swap! state assoc :server-error (get-parsed-response false))))}))
+   :get-ordered-columns
+   (fn [{:keys [refs]}]
+     ((@refs "table") :get-ordered-columns))
```
Comment (MattPutnam) [inline src/cljs/main/broadfcui/common/entity_table.cljs]: This is unused now, right?

File: src/cljs/main/broadfcui/common/table/table.cljs [hunk 4]
```diff
@@ -70,6 +70,9 @@
                                        :filtered-count filtered-count
                                        :rows results
                                        :query-params query-params))})))
+   :get-ordered-columns
+   (fn [{:keys [state]}]
+     (:column-display @state))
```
Comment (MattPutnam) [inline src/cljs/main/broadfcui/common/table/table.cljs]: Same here?

File: src/cljs/main/broadfcui/common/table/table.cljs [hunk 5]
```diff
@@ -1,17 +1,17 @@
...
+    [clojure.string :as string]
+    [broadfcui.common.components :as comps]
+    [broadfcui.common.style :as style]
+    [broadfcui.common.table.body :as body]
+    [broadfcui.common.table.column-editor :refer [ColumnEditButton]]
+    [broadfcui.common.table.paginator :refer [Paginator]]
+    [broadfcui.common.table.utils :as table-utils]
+    [broadfcui.persistence :as persistence]
+    [broadfcui.utils :as utils]
+    ))
```
Comment (MattPutnam) [inline src/cljs/main/broadfcui/common/table/table.cljs]: We(Isaac and I)'re not sure what changed, but now the one space indentation (what's in red here) is right.

**Review by MattPutnam — APPROVED** [review]
👍 with some cleanup

File: src/cljs/main/broadfcui/common/entity_table.cljs [hunk 6]
```diff
@@ -113,22 +116,25 @@
...
-                              :selected-index (:selected-filter-index @state)
-                              :on-change (fn [index _]
-                                           (let [selected-entity-type (nth entity-types index)]
-                                             (swap! state assoc
-                                                    :selected-filter-index index
-                                                    :selected-entity-type selected-entity-type)
-                                             (when-let [f (:on-entity-type-selected props)]
-                                               (f selected-entity-type))))}]
-                            (:toolbar-items props))
+              {:items
```
Comment (dmohs) [inline src/cljs/main/broadfcui/common/entity_table.cljs:119]: This should probably be a follow-on PR and it's my fault, but `items` is no longer an appropriate name for this key. It's a function, so it should start with a verb. I suggest `get-items`.

File: src/cljs/main/broadfcui/common/entity_table.cljs [hunk 7]
```diff
@@ -113,22 +116,25 @@
...
+                                            entity-types)
+                        :selected-index (:selected-filter-index @state)
+                        :on-change (fn [index _]
+                                     (let [selected-entity-type (nth entity-types index)]
+                                       (swap! state assoc
+                                              :selected-filter-index index
+                                              :selected-entity-type selected-entity-type)
+                                       (when-let [f (:on-entity-type-selected props)]
+                                         (f selected-entity-type))))}]
+                      ((:toolbar-items props) table-props)))
```
Comment (dmohs) [inline src/cljs/main/broadfcui/common/entity_table.cljs:134]: Likewise, I suggest `get-toolbar-items`.

**Review by dmohs — APPROVED** [review]
Looks good, but we should also do the renames soon to keep this code from degrading in terms of maintainability.

**Pull request merged**